src/components/ConfirmedRoute.tsx

```diff
@@ -90,19 +79,21 @@ function ConfirmedRoute({mapboxAccessToken, transaction}) {
 
     useEffect(() => {
         MapboxToken.init();
-        return MapboxToken.stop;
+        return () => {
+            MapboxToken.stop();
+        };
```

Comment:
Why was this changed?
```suggestion
        return MapboxToken.stop
```